### PR TITLE
[#6643] Allow chat command suggestions to select parameter

### LINKED ISSUE #6643: Allow chat command suggestions to select parameter
### User Story

As a developer, I want to be able to trigger parameter selection from a chat command suggestion view.

### Description

*Type*: Feature

Proper syntax should be defined first.

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #6643

added new `chat.command/set-parameter` event

Usage in extension

```
 views/suggestion
 [view {:style {:flex-direction :row}
               :align-items    :flex-start}
  [button {:on-click [chat.command/set-parameter {:value "Yohoho"}]}]]
```

### CODE REVIEW

File: src/status_im/extensions/core.cljs [hunk 1]
```diff
@@ -206,17 +211,19 @@
                 'link               {:value link :properties {:uri :string}}
                 ;'list               {:value list :properties {:data :vector :item-view :view}}
                 'checkbox           {:value checkbox :properties {:on-change :event :checked :boolean}}
-                'nft-token-viewer   {:value transactions/nft-token :properties {:token :string}}
-                'transaction-status {:value transactions/transaction-status :properties {:outgoing :string :tx-hash :string}}
-                'asset-selector     {:value transactions/choose-nft-asset-suggestion}
-                'token-selector     {:value transactions/choose-nft-token-suggestion}}
+                'asset-selector     {:value #()}
```
Comment (flexsurfer) [inline src/status_im/extensions/core.cljs]: this is by mistake

File: src/status_im/extensions/core.cljs [hunk 2]
```diff
@@ -207,16 +212,18 @@
...
-                'token-selector     {:value transactions/choose-nft-token-suggestion}}
+                'transaction-status {:value transactions/transaction-status :properties {:outgoing :string :tx-hash :string}}}
    :queries    {'identity            {:value :extensions/identity :arguments {:value :map}}
                 'store/get           {:value :store/get :arguments {:key :string}}
                 'wallet/collectibles {:value :get-collectible-token :arguments {:token :string :symbol :string}}}
    :events     {'alert
                 {:permissions [:read]
                  :value       :alert
                  :arguments   {:value :string}}
+                'select-suggestion
```
Comment (flexsurfer) [inline src/status_im/extensions/core.cljs]: or maybe better `set-parameter`

**Review by goranjovic — APPROVED** [review]

File: src/status_im/extensions/core.cljs (hunk 2 above)
Comment (jeluard) [reply src/status_im/extensions/core.cljs]: Maybe `chat.command/set-parameter` ?

**Review by jeluard — APPROVED** [review]

**Comment by jeluard** [comment]
@flexsurfer Please rebase :)

**Comment by asemiankevich** [comment]
@jeluard @antdanchenko should be fine to merge?

**Pull request merged**